cmd/kubeadm/app/util/staticpod/utils.go

```diff
@@ -29,19 +28,21 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/google/go-cmp/cmp"
 	"github.com/pkg/errors"
 
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/dump"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/klog/v2"
 
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	kubeadmutil "k8s.io/kubernetes/cmd/kubeadm/app/util"
 	"k8s.io/kubernetes/cmd/kubeadm/app/util/patches"
 	"k8s.io/kubernetes/cmd/kubeadm/app/util/users"
+	"k8s.io/kubernetes/pkg/util/hash"
```

Comment:
k8s.io/kubernetes/pkg is for internal kuberentes use only. kubeadm cannot import that.

Reply:
Thanks. Just realize that. 

I update the pr to show the pod diff.